packages/utils/src/lib/report.unit.test.ts

```diff
@@ -182,3 +190,104 @@ describe('loadReport', () => {
+      { slug: 'a4', weight: 0, plugin: 'a', score: 0, value: 1, title: 'A4' },
+    ];
+    const sortedAudits = [...mockAudits].sort(sortCategoryAudits);
+    expect(sortedAudits[0]).toEqual(mockAudits[3]);
+    expect(sortedAudits[1]).toEqual(mockAudits[0]);
+    expect(sortedAudits[2]).toEqual(mockAudits[2]);
+    expect(sortedAudits[3]).toEqual(mockAudits[1]);
+  });
+
+  it('should sort audits by value and title', () => {
+    const mockAudits: WeighedAuditReport[] = [
+      { slug: 'a1', weight: 0, plugin: 'a', score: 0.1, value: 1, title: 'c' },
+      { slug: 'a2', weight: 0, plugin: 'a', score: 0.1, value: 0, title: 'b' },
+      { slug: 'a3', weight: 0, plugin: 'a', score: 0.1, value: 0, title: 'a' },
+      { slug: 'a4', weight: 0, plugin: 'a', score: 0.1, value: 1, title: 'd' },
+    ];
+    const sortedAudits = [...mockAudits].sort(sortCategoryAudits);
+    expect(sortedAudits[0]).toEqual(mockAudits[0]);
+    expect(sortedAudits[1]).toEqual(mockAudits[3]);
+    expect(sortedAudits[2]).toEqual(mockAudits[2]);
+    expect(sortedAudits[3]).toEqual(mockAudits[1]);
+  });
+});
+
+describe('sortAudits', () => {
+  it('should sort audits by score and value', () => {
+    const mockAudits: EnrichedAuditReport[] = [
+      { slug: 'a1', plugin: 'a', score: 0.7, value: 1, title: 'A1' },
+      { slug: 'a2', plugin: 'a', score: 1, value: 1, title: 'A2' },
+      { slug: 'a3', plugin: 'a', score: 0.7, value: 0, title: 'A3' },
+      { slug: 'a4', plugin: 'a', score: 0, value: 1, title: 'A4' },
+    ];
+    const sortedAudits = [...mockAudits].sort(sortAudits);
+    expect(sortedAudits[0]).toEqual(mockAudits[3]);
+    expect(sortedAudits[1]).toEqual(mockAudits[0]);
+    expect(sortedAudits[2]).toEqual(mockAudits[2]);
+    expect(sortedAudits[3]).toEqual(mockAudits[1]);
+  });
+
+  it('should sort audits by value and title', () => {
+    const mockAudits: EnrichedAuditReport[] = [
+      { slug: 'a1', plugin: 'a', score: 0.1, value: 1, title: 'c' },
+      { slug: 'a2', plugin: 'a', score: 0.1, value: 0, title: 'b' },
+      { slug: 'a3', plugin: 'a', score: 0.1, value: 0, title: 'a' },
+      { slug: 'a4', plugin: 'a', score: 0.1, value: 1, title: 'd' },
+    ];
+    const sortedAudits = [...mockAudits].sort(sortAudits);
+    expect(sortedAudits[0]).toEqual(mockAudits[0]);
+    expect(sortedAudits[1]).toEqual(mockAudits[3]);
+    expect(sortedAudits[2]).toEqual(mockAudits[2]);
+    expect(sortedAudits[3]).toEqual(mockAudits[1]);
+  });
+});
+
+describe('getPluginNameFromSlug', () => {
+  it('should return plugin name', () => {
+    const plugins: ScoredReport['plugins'] = [
+      {
+        slug: 'plugin-a',
+        title: 'Plugin A',
+        date: '',
+        duration: 0,
+        icon: 'todo',
+        audits: [],
+        groups: [],
+      },
+      {
+        slug: 'plugin-b',
+        title: 'Plugin B',
+        date: '',
+        duration: 0,
+        icon: 'todo',
+        audits: [],
+        groups: [],
+      },
+    ];
```

Comment:
A small trick I like to do in tests (only in tests, though!) is to do a soft type-cast that lets me omit properties which are required under usual circumstances, but aren't relevant to the test. The advantage is that it removes the extra noise from the test.

```suggestion
    const plugins = [
      {
        slug: 'plugin-a',
        title: 'Plugin A',
      },
      {
        slug: 'plugin-b',
        title: 'Plugin B',
      },
    ] as ScoredReport['plugins'];
```

This is just a suggestion, though. It's still a small test, so the extra noise doesn't matter so much.

Reply:
Awesome hack! Implemented for all new tests, as they also contain mocks